ChangeLog

```diff
@@ -422,7 +422,8 @@ and bug requests.
    * filter.d/uwimap-auth - filter for uwimap-auth IMAP/POP server
   Steven Hiscocks and Daniel Black
    * filter.d/selinux-{common,ssh} -- add SELinux date and ssh filter
-
+  Donald Yandt
+   * action.d/firewallcmd-{multiport,allports}.conf --added two new actions
```

Comment:
Sorry, I have a serious flu right now so I'm not thinking straight. I'll try and get everything straightened out.